extensions/canvasMedical/v1/client/__mocks__/testData.ts

```diff
@@ -505,6 +505,97 @@ export const mockedQuestionnaireResponseResource: QuestionnaireResponseWithId =
     ...mockedQuestionnaireResponseData,
   }
 
+export const mockedCreateCoverageData: any = {
+  order: 1,
+  status: 'active',
+  type: JSON.stringify({
+    coding: [
+      {
+        system: 'http://hl7.org/fhir/ValueSet/coverage-type',
+        code: 'MILITARY',
+        display: 'military health program',
+      },
+    ],
+    text: 'Military',
+  }),
+  subscriber: 'Patient/01e226b6ad324595a4b7030d003db06f',
+  subscriberId: '1234',
+  beneficiary: 'Patient/01e226b6ad324595a4b7030d003db06f',
+  relationship: JSON.stringify({
+    coding: [
+      {
+        system: 'http://hl7.org/fhir/ValueSet/subscriber-relationship',
+        code: 'self',
+      },
+    ],
+  }),
+  periodStart: '2021-06-27',
+  periodEnd: '2022-06-27',
+  payor: JSON.stringify([
+    {
+      identifier: {
+        system: 'https://www.claim.md/services/era/',
+        value: 'AMM03',
+      },
+      display: 'Independence Blue Cross Blue Shield',
+    },
+  ]),
+  classObj: JSON.stringify([
+    {
+      type: {
+        coding: [
+          {
+            system: 'http://hl7.org/fhir/ValueSet/coverage-class',
+            code: 'group',
+          },
+        ],
+      },
+      value: 'Captains Only',
+    },
+    {
+      type: {
+        coding: [
+          {
+            system: 'http://hl7.org/fhir/ValueSet/coverage-class',
+            code: 'subgroup',
+          },
+        ],
+      },
+      value: 'Subgroup 2',
+    },
+    {
+      type: {
+        coding: [
+          {
+            system: 'http://hl7.org/fhir/ValueSet/coverage-class',
+            code: 'plan',
+          },
+        ],
+      },
+      value: 'Starfleet HMO',
+    },
+    {
+      type: {
+        coding: [
+          {
+            system: 'http://hl7.org/fhir/ValueSet/coverage-class',
```

Comment:
Use this link for the coverage class above
Also if this is coverage class then can you rename the label for the field to that (Coverage Class)